pkg/apis/config/v1alpha1/interpretercontext_types.go

```diff
@@ -153,6 +153,12 @@ type DependentObjectReference struct {
 	Namespace string `json:"namespace,omitempty"`
 
 	// Name represents the name of the referent.
-	// +required
-	Name string `json:"name"`
+	// +optional
+	Name string `json:"name,omitempty"`
+
+	// A label query over a set of resources.
+	// If name is not empty, labelSelector will be ignored.
+	// Name and LabelSelector cannot be empty at the same time.
```

Comment:
Are we planning to add corresponding return value verification in another PR?

Reply:
> Are we planning to add corresponding return value verification in another PR?

Maybe, I need to do some research.